notebook/notebook/notebook.py

```diff
@@ -56,15 +57,17 @@ def read_string(f):
 log.info(f'INSTANCE_ID {INSTANCE_ID}')
 
 try:
-    with open('notebook-worker-image', 'r') as f:
-        WORKER_IMAGE = f.read().strip()
+    with open('notebook-worker-images', 'r') as f:
+        def get_name(line):
+            return re.search("/[^/:]+:", line).group()[1:-1]
```

Comment:
What does `group` do with no arguments? It doesn't look like that regex has any groups? This is what I would've expected: `re.search("/([^/:]+):", line).group(1)`

Reply:
dunno, this worked though. Your way is better